device/celestica/x86_64-cel_e1031-r0/sonic_platform/common.py

```diff
@@ -0,0 +1,283 @@
+
+        if type(argument) is list:
+            sysfs_path = sysfs_path.format(argument[index])
+
+        write_offset = int(config.get('write_offset', 0))
+        output = ""
+        try:
+            open_file = open(sysfs_path, "r+")
+            open_file.seek(write_offset)
+            open_file.write(input)
+            open_file.close()
+        except IOError as e:
+            print("Error: unable to open file: %s" % str(e))
+            return False, output
+        return True, output
+
+    def _ipmi_set(self, index, config, input):
+        arg = config['argument'][index].format(input)
+        return self.run_command(config['command'].format(arg))
+
+    def _hex_ver_decode(self, hver, num_of_bits, num_of_points):
+        ver_list = []
+        c_bit = 0
+        bin_val = bin(int(hver, 16))[2:].zfill(num_of_bits)
+        bit_split = num_of_bits / (num_of_points + 1)
+        for x in range(0, num_of_points+1):
+            split_bin = bin_val[c_bit:c_bit+bit_split]
+            ver_list.append(str(int(split_bin, 2)))
+            c_bit += bit_split
+        return '.'.join(ver_list)
+
+    def _get_class(self, config):
+        """
+        Retreives value of expected attribute
+        Returns:
+            A value of the attribute of object
+        """
+        path = config['host_path'] if self.is_host() else config['pmon_path']
+        module = imp.load_source(config['class'], path)
+        class_ = getattr(module, config['class'])
+        return class_
+
+    def get_reg(self, path, reg_addr):
+        cmd = "echo {1} > {0}; cat {0}".format(path, reg_addr)
+        status, output = self.run_command(cmd)
+        return output if status else None
+
+    def read_txt_file(self, path):
+        with open(path, 'r') as f:
+            output = f.readline()
+        return output.strip('\n')
+
+    def write_txt_file(self, file_path, value):
+        try:
+            with open(file_path, 'w') as fd:
+                fd.write(str(value))
+        except Exception:
+            return False
+        return True
+
+    def is_host(self):
+        return os.system(self.HOST_CHK_CMD) == 0
+
+    def load_json_file(self, path):
+        """
+        Retrieves the json object from json file path
+
+        Returns:
+            A json object
+        """
+        with open(path, 'r') as f:
+            json_data = yaml.safe_load(f)
+
+        return json_data
+
+    def get_config_path(self, config_name):
+        """
+        Retrieves the path to platform api config directory
+
+        Args:
+            config_name: A string containing the name of config file.
+
+        Returns:
+            A string containing the path to json file
+        """
+        return os.path.join(self.DEVICE_PATH, self.platform, self.CONFIG_DIR, config_name) if self.is_host() else os.path.join(self.PMON_PLATFORM_PATH, self.CONFIG_DIR, config_name)
+
+    def get_output(self, index, config, default):

```

Comment:
>get_output [](http://example.com/codeflow?start=8&length=10)

Seems this function is not used. Please remove.
@mudsut4ke Could you help review?